series/templates/series/show_details.html

```diff
@@ -73,14 +73,38 @@ <h1>{{ show.title }}</h1>
         </div>
     </div>
 
-    {% for season in show.seasons reversed %}
-        <p>Season {{ season.number }}</p>
-        <ul>
+    <ul id="season-nav">
+        {% for season_number in seasons_range %}
+            <li {% if season and season.number == season_number %}class="active"{% endif %}>
+                <a href="?season={{ season_number }}">
+                    Season {{ season_number }}
+                </a>
+            </li>
+        {% endfor %}
+    </ul>
+
+    {% if season %}
+        <ul id="season-episodes">
             {% for episode in season.episodes reversed %}
-                <li> Episode {{ episode.number }}: {{ episode.synopsis }} </li>
+                <li>
+                    <img src="{{ episode.still_path }}">
+                    <div class="body">
+                        <p>
+                            <strong>
+                                Episode {{ episode.number }}
```

Comment:
I realize I'm the one who didnt think about it in the first place, but it would be great to add the episode name too.